packages/grafana-data/src/types/logsVolume.ts

```diff
@@ -0,0 +1,19 @@
+import { DataQuery } from './query';
+import { DataQueryRequest, DataQueryResponse } from './datasource';
+import { Observable } from 'rxjs';
+
+/**
+ * TODO: This should be added to ./logs.ts but because of cross reference between ./datasource.ts and ./logs.ts it can
+ * be done only after decoupling "logs" from "datasource" (https://github.com/grafana/grafana/pull/39536)
+ *
+ * @internal
+ */
```

Comment:
@aocenas :

>  The data itself seems to be wrong. This is the same query the smaller graph is actually later. When I am refreshing it seems to oscillate between some smaller and bigger value seemingly randomly. I would expect it to either stay the same or grow.

> Also the Y axis I would assume should be adjusted instead of going from 0 - 100 if there is only few logs.

Hmmm... this is automatically adjusted by time series panel and I can't make it look as bad as on your screenshot :D Is it something easy to reproduce with gdev-loki or TNS?

Reply:
Possibly related to running instant queries. This should work better with f5fc283dbe79d315d90fa3809239fdfea9488ac6 and will be revised in #39856.